common/test/acceptance/pages/lms/discussion.py

```diff
@@ -688,6 +688,26 @@ def perform_search(self, text="dummy"):
             "waiting for server to return result"
         ).fulfill()
 
+    def is_element_visible(self, selector):
+        """
+        Returns true if the element matching the specified selector is visible.
+        """
+        query = self.q(css=selector)
+        return query.present and query.visible
+
+    def is_checkbox_selected(self):
+        """
+        Returns true or false if the 'Receive update' checkbox is selected or not .
+        """
```

Comment:
Why only for 'Receive update'? We may have a generic method here that may be used in future for some other case. You may pass `selector` here to be passed to `self.q()`.